tark/tark/templates/documentation.html

```diff
@@ -1,11 +1,91 @@
+<head>
+	<style>
+	<!---------------------------------------------------------------------------------->
+	<!--------------------------------Add CSS for collapsible------------------>
+	<!---------------------------------------------------------------------------------->
+	.collapsible {
+		  background-color: #fff;
+		  color: #333;
+		  cursor: pointer;
+		  padding: 5px;
+		  width: 100%;
+		  border: #ddd;
+		  text-align: left;
+		  outline: none;
+		  <!--font-size: 15px; -->
+	}
+	.active-expand, .collapsible:hover {
+		  background-color: #337ab7;
+		  color: #fff;
+	}
+	.collapsible:after {
+		  content: '\002B';;  <!--plus symbol after collapse-->
+		  color: #337ab7;
+		  font-weight: bold;
+		  float: right;
+		  margin-left: 5px;
+	}
+	.active-expand:after {
+		  content: "\2212";; <!--minus symbol after expanding-->
+	}
+	.content {
+		  padding: 0 18px;
+		  max-height: 0;
+		  overflow: hidden;
+		  transition: max-height 0.2s ease-out;
+		  background-color: #f1f1f1;
+	}
+	<!---------------------------------------------------------------------------------->
```

Comment:
As mentioned above:
- `<!--` [are not recommended in CSS](https://developer.mozilla.org/en-US/docs/Web/CSS/Comments#notes) and result in compilation errors, use `/* */` instead.
- One simple line is enough (e.g. `/* Add CSS for collapsible */`)